client/jetpack-app/plans/style.scss

```diff
@@ -0,0 +1,33 @@
+@import "calypso/my-sites/plans-grid/media-queries";
+
+body {
+	background-color: #fdfdfd;
```

Comment:
I there a CSS variable we can use for this?
https://wpcalypso.wordpress.com/devdocs/docs/color.md

Maybe  --studio-gray-0? (#f6f7f7)

Reply:
Good question. This one is a bit too dark, unfortunately. I'm using the color to fit the Jetpack app colors. However, if this would be a blocker we could live with that for now 👍

Reply:
Yes, I ended up setting the color as `--studio-white`. This is what works for us.